main.go

```diff
@@ -0,0 +1,502 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/bitrise-io/go-utils/colorstring"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/pathutil"
+	"github.com/bitrise-io/go-utils/stringutil"
+	"github.com/bitrise-io/steps-xcode-archive/utils"
+	"github.com/bitrise-steplib/steps-xcode-build-for-simulator/util"
+	"github.com/bitrise-tools/go-steputils/stepconf"
+	"github.com/bitrise-tools/go-xcode/simulator"
+	"github.com/bitrise-tools/go-xcode/utility"
+	"github.com/bitrise-tools/go-xcode/xcodebuild"
+	"github.com/bitrise-tools/go-xcode/xcpretty"
+	"github.com/bitrise-tools/xcode-project/xcodeproj"
+	"github.com/bitrise-tools/xcode-project/xcscheme"
+	"github.com/bitrise-tools/xcode-project/xcworkspace"
+	shellquote "github.com/kballard/go-shellquote"
+)
+
+const (
+	minSupportedXcodeMajorVersion = 6
+	iOSSimName                    = "iphonesimulator"
+	tvOSSimName                   = "appletvsimulator"
+	watchOSSimName                = "watchsimulator"
+)
+
+const (
+	bitriseXcodeRawResultTextEnvKey = "BITRISE_XCODE_RAW_RESULT_TEXT_PATH"
```

Comment:
test results text used to be report, BITRISE_XCODE_TEST_RESULT_PATH?

Reply:
Yes, but we are not running tests with this step. Only building an app.
But if the user uses the `xcpretty` output tool, then it will export the `raw log` to this path.

Just like in the Xcode Archive:
https://github.com/bitrise-io/steps-xcode-archive/blob/master/main.go#L39

Reply:
Agree with @BirmacherAkos